src/components/files/routeArchiveFiles.ts

```diff
@@ -0,0 +1,15 @@
+import { FastifyReply, FastifyRequest } from 'fastify'
+import { combineRequestParams } from '../utilityFunctions'
+import archiveFiles from './archive'
+
+const routeArchiveFiles = async (request: FastifyRequest, reply: FastifyReply) => {
```

Comment:
Added an endpoint for this so Archives can be triggered by front-end (on Snapshots page)